publish.gradle

```diff
@@ -13,10 +13,13 @@ def libVcsUrl = properties.libVcsUrl
 // please also update the corresponding .buildconfig-android.yml
 // `publishedArtifacts` property.
 ext.configurePublish = { jnaForTestConfiguration = null,
-                         variantWithoutLib = null ->
+                         isMegazord = false ->
     def theGroupId = rootProject.ext.library.groupId
     def theArtifactId = project.ext.artifactId
     def theDescription = project.ext.description
+
+    // XXX update this comment once things work.
+    //
```

Comment:
Lol, do things work well enough for this comment to be updated now?

Reply:
Yes, it and various other comments are all updated (Could have missed some).

Honestly I think I'm probably going to write a slightly longer document that at least outlines the important points about the way things work now, since it's not that complex but some bits of it are subtle.